pom.xml

```diff
@@ -448,6 +448,56 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>owasp</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.owasp</groupId>
+                        <artifactId>dependency-check-maven</artifactId>
+                        <version>1.2.9</version>
+                        <configuration>
+                            <failBuildOnCVSS>7</failBuildOnCVSS>
+                            <format>ALL</format>
+                            <suppressionFile>dependency-check-maven-config.xml</suppressionFile>
```

Comment:
I'm thinking no because the intent of this PR is to provide the ability for OWASP to be run and vulnerabilities to be found. As soon as this PR is merged, I plan to make a ticket to fix the particular vulnerabilities that are found by running OWASP and I can include making a new dependency-check-maven-config.xml in the description.